postProcessing/Folders/CosmicIntegration/PythonScripts/ClassCosmicIntegrator.py

```diff
@@ -87,9 +88,16 @@ def __init__(self,pathCOMPAS=None, Cosmology='WMAP',hubbleConstant = 67.8,\
         if self.verbose:
             print("Creating instance COMPAS class User has to still set DCO and Data")
         #setting Mlower/Mupper etc to None to force warning for user
```

Comment:
@themikelau You can do the "if fileName is None" part more succinctly. In the __init__ call, use `fileName=COMPAS_output.h5` or whatever the default is.

Reply:
The idea here is that ClassCosmicIntegrator will use the default filename in ClassCOMPAS if no filename is given. This has the advantage of not having to maintain and keep consistent the two default filenames in the two classes, but indeed at the cost of succinctness. I can try to think about how to shorten this code.

Reply:
I understand, but python has the functionality you're describing built-in.

```
myFunc(fileName=None):
	if fileName is None:
		doSomething(defaultFileName)
	else:
		doSomething(fileName)
```
Is the same as 
```
myFunc(fileName=defaultFileName):
	doSomething(fileName)
```

Reply:
The correct pseudo-code that describes what is currently implemented is
```
myFunc(fileName=None):
	if fileName is None:
		doSomething()
	else:
		doSomething(fileName)
```
which is different from 
```
myFunc(fileName=defaultFileName):
	doSomething(fileName)
```
because in the first case, when no fileName is given to myFunc, the filename that ends up getting used is the default filename in doSomething, whereas in the second case, the filename getting used is the default filename in myFunc.